src/challonge.service.js

```diff
@@ -1,121 +1,119 @@
 const R = require('ramda');
 
-function challongeServiceFactory({ api, organization }) {
-    async function fetchAllTournaments() {
-        const { data } = await api.get('tournaments.json', {
-            params: {
-                subdomain: organization,
-            },
-        });
-        return R.map(({ tournament }) => tournament)(data);
-    }
+const fetchAllTournaments = ({ api, organization }) => async function () {
+    const { data } = await api.get('tournaments.json', {
+        params: {
+            subdomain: organization,
+        },
+    });
+    return R.map(({ tournament }) => tournament)(data);
+}
 
-    async function fetchOpenTournaments() {
-        const { data } = await api.get('tournaments.json', {
-            params: {
-                subdomain: organization,
-                // state: '...', // one of 'all', 'pending', 'in_progress', 'ended'
-            },
-        });
-        return R.pipe(
-            R.map(({ tournament }) => tournament),
-            R.filter(t => ['pending', 'underway'].includes(t.state)),
-        )(data);
-    }
+const fetchOpenTournaments = ({ api, organization }) => async function () {
+    const { data } = await api.get('tournaments.json', {
+        params: {
+            subdomain: organization,
+            // state: '...', // one of 'all', 'pending', 'in_progress', 'ended'
+        },
+    });
+    return R.pipe(
+        R.map(({ tournament }) => tournament),
+        R.filter(t => ['pending', 'underway'].includes(t.state)),
+    )(data);
+}
 
-    async function fetchTournament(tournamentId) {
-        const { data } = await api.get(`tournaments/${tournamentId}.json`, {
-            params: {
-                include_participants: 1,
-                include_matches: 1,
-            }
-        });
-        const { tournament } = data;
-        return {
-            ...tournament,
-            participants: R.map(({ participant }) => participant)(tournament.participants),
-            matches: R.map(({ match }) => match)(tournament.matches),
-        };
-    }
+const fetchTournament = ({ api }) => async function (tournamentId) {
```

Comment:
Okay, I see the beauty now of injecting an object rather than separate params. :beach_umbrella: :tada: